### PR TITLE
Age gender detect

### PR DESCRIPTION
**Pull request by jhalak27**
For issue #24 
Added age and gender detection

### CODE REVIEW

**Comment by smaranjitghose** [comment]
Nice work jhalak!.. For the next part of the issue can you please take an opensource dataset and use transfer learning approach like Efficientnet or MobileNetv3 to train the models in Tensorflow(even PyTorch or fastai is cool) for higher accuracy
